src/fxcorrelator.py

```diff
@@ -992,8 +1001,9 @@ def _handle_sources(self):
         for counter, address in enumerate(source_mcast):
             new_source = DataSource.from_mcast_string(address)
             new_source.name = source_names[counter]
+            # adding a new instance attribute here, be careful
             new_source.source_number = source_ctr
-            assert new_source.iprange == self.ports_per_fengine, \
+            assert new_source.ip_range == self.ports_per_fengine, \
```

Comment:
Could write as 

    assert new_source.ip_range == self.ports_per_fengine, (
        'F-engines shoul ......')

instead of using \ line continuation. Super-minor style nitpick :)